### ISSUE TITLE
[CODAP] Is the 'Field' variable needed for PSD?

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/projectile-data-lab#344]
For https://github.com/phetsims/qa/issues/1099;
There aren't different fields in PSD but there is a Field checkbox. In the data table the Field column lists the launcher and the sample size. Wondering if the Field variable checkbox is needed since the Launcher and Sample Size have their own checkboxes or if it is needed, should it have a different name?

<img width="627" alt="Screenshot 2024-06-20 at 1 05 32 PM" src="https://github.com/phetsims/projectile-data-lab/assets/87318828/561e9627-94c6-44d3-9453-008559483b8e">

**Comment by matthew-blackman** [comment]
@catherinecarter interested in your thoughts. Do you feel that Field can be removed for the PSD table, since that info is available through a combination of Launcher and Sample Size? Or is it more convenient in CODAP to have them wrapped up in a unique identifier for each independent data set?

**Comment by catherinecarter** [comment]
I tested it out a bit, and having them separate is actually better. It's more clear to the user when checking the checkboxes for launcher and sample size, that they aren't wrapped together under the word, "Field," which is not as intuitive as a label.

I think Field can be removed, as long as the launcher and sample size are still available as variables to choose. 

I do think that if the Launcher checkbox is defaulted to be checked, the sample size should also be defaulted to checked since they go hand-in-hand. Which is what the current Field checkbox is currently intended to do, I believe.

In summary: 
- Yes, delete Field
- Default checkboxes: 
  - Sample size
  - Launcher

**Comment by catherinecarter** [comment]
Adding another comment: 

We will need to improve the Help dialog box. The pathway to get a graph of the sample size/launcher combos, while also calculating the mean per sample is quite a bear (see image below). I'll brainstorm ideas.
![image](https://github.com/user-attachments/assets/a0285c9c-f263-41f9-a5f8-a489865e2671)

**Comment by matthew-blackman** [comment]
@catherinecarter this is ready for review. I'll make a separate issue for the Help dialog on PSD.

**Comment by catherinecarter** [comment]
I'm still seeing the Field and caseIndex columns, and the sample number/size are unchecked. I was checking dev.3. Am I in the correct version?

**Comment by catherinecarter** [comment]
Looks great. Much cleaner and easier to parse.
